Web/DpAssignment1/designpattern1qn2/Tester.java

```diff
@@ -0,0 +1,66 @@
+package designpattern1qn2;
+
+public class Tester implements IEngineer {
```

Comment:
Tester and Developer looks pretty same with at least same member variables and methods. You should have shown some difference :D